README.md

```diff
@@ -80,9 +80,9 @@ binaries, including OpenROAD, Yosys and Klayout. See instructions
 > **Disclaimer** The versions of OpenROAD, Yosys and Klayout provided by
 > other third-party vendors are not guaranteed to work with ORFS.
 
-### Local Installation
+### Build from sources locally
```

Comment:
@tspyrou @maliberty This is ambigous. You build docker images and local install both from local source files.